modules/log-upgrade/6x-logging-upgrading-clo.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * observability/logging/cluster-logging-upgrading.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="logging-upgrading-clo_{context}"]
+= Updating the {clo}
+
+To update the {clo} to a new major release version, you must modify the update channel for the Operator subscription.
```

Comment:
It's not indicated how to create the new CR with the observability API. Upgrading to Logging 6 is not only upgrading the new operator. This manual "migration" is what's tried to be explained in https://docs.redhat.com/en/documentation/openshift_container_platform/4.16/html/logging/logging-6-0 points  2.4.5, 2.4.6, 2.4.7...2.4.11, etc. Basically, what's needed to know before doing any step for upgrading is how to transform the current configuration to Logging 6. The changes are detailed in - https://github.com/openshift/cluster-logging-operator/blob/master/docs/administration/upgrade/v6.0_changes.adoc